addon/bootstrap.js

```diff
@@ -59,7 +61,59 @@ const appStartupObserver = {
   }
 }
 
+const LibraryButton = {
+  ITEM_ID: "appMenu-library-screenshots",
+
+  init(webExtension) {
+    this._initialized = true;
+    let permissionPages = [...webExtension.extension.permissions].filter(p => (/^https?:\/\//i).test(p));
+    this.PAGE_TO_OPEN = permissionPages.length ? permissionPages[0].replace(/\*$/, "") : "https://screenshots.firefox.com/";
+    this.PAGE_TO_OPEN += "shots";
+    this.ICON_URL = webExtension.extension.getURL("icons/icon-16-v2.svg");
+    this.ICON_URL_2X = webExtension.extension.getURL("icons/icon-32-v2.svg");
```

Comment:
I'm not sure how the photon library UI works (I haven't seen mocks or specs), but right now, in the toolbar button, we [show a special icon](https://github.com/mozilla-services/screenshots/blob/9959edeff2b9c8a91916ad63037cecc92438860e/addon/webextension/background/startBackground.js#L55-L60) to users who haven't tried Screenshots before, and we [swap in the regular icon](https://github.com/mozilla-services/screenshots/blob/9959edeff2b9c8a91916ad63037cecc92438860e/addon/webextension/background/main.js#L53-L57) after they complete onboarding. (Note that startBackground runs first, lazy-loading the rest of the webextension, including main.js.)

If this UI is replacing the toolbar button UI, I think we'd want to keep that special icon behavior for new users.

If this item is in addition to some photon-specific "take a shot" UI, I'm not sure what the flow should be. If a user hasn't tried Screenshots before, I think we'd want to show them the onboarding flow, and maybe not open a link to the shots page. @johngruen, thoughts?

I'm also not sure we'd want to reuse the same icon for multiple commands related to the same feature (take a shot vs visit shots page).